django/contrib/gis/db/backends/spatialite/operations.py

```diff
@@ -123,12 +121,13 @@ def spatial_version(self):
         """Determine the version of the SpatiaLite library."""
         try:
             version = self.spatialite_version_tuple()[1:]
-        except Exception as msg:
-            new_msg = (
-                'Cannot determine the SpatiaLite version for the "%s" '
-                'database (error was "%s").  Was the SpatiaLite initialization '
-                'SQL loaded on this database?') % (self.connection.settings_dict['NAME'], msg)
-            six.reraise(ImproperlyConfigured, ImproperlyConfigured(new_msg), sys.exc_info()[2])
+        except Exception:
+            raise ImproperlyConfigured(
+                'Cannot determine the SpatiaLite version for the "%s" database. '
+                'Was the SpatiaLite initialization SQL loaded on this database?' % (
```

Comment:
Unrelated: this might be obsolete after 8f97413faed5431713c034897cda486507bf0cc3.